<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationControl</class>
 <widget class="QWidget" name="AnimationControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Current Animation:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" colspan="5">
       <widget class="QLabel" name="lAnimationCurrentName">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Animation</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="pbFPS1">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="6">
       <widget class="QComboBox" name="cbAnimation"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="5">
       <widget class="QLabel" name="lAniamtionFPS">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Animation</string>
        </property>
       </widget>
      </item>
      <item row="7" column="6">
       <widget class="QPushButton" name="pbFPS4">
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="pbFPS2">
        <property name="text">
         <string>25</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="7">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>State:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Animation:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>11</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="0" colspan="7">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="7">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="cbSync">
          <property name="text">
           <string>Sync on Change</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSet">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSave">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2" colspan="5">
       <widget class="QLabel" name="lAnimationState">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Animation</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QPushButton" name="pbFPS3">
        <property name="text">
         <string>40</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5" colspan="2">
       <widget class="QPushButton" name="pbStop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="4">
       <widget class="QPushButton" name="pbAnimate">
        <property name="text">
         <string>Animate</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>fps</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>cbAnimation</zorder>
     <zorder>pbAnimate</zorder>
     <zorder>pbStop</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>lAnimationCurrentName</zorder>
     <zorder>label_4</zorder>
     <zorder>lAnimationState</zorder>
     <zorder>label_3</zorder>
     <zorder>lAniamtionFPS</zorder>
     <zorder>spinBox</zorder>
     <zorder>label_5</zorder>
     <zorder>verticalSpacer</zorder>
     <zorder>verticalSpacer_2</zorder>
     <zorder>line</zorder>
     <zorder>line_2</zorder>
     <zorder>pbFPS4</zorder>
     <zorder>pbFPS1</zorder>
     <zorder>pbFPS2</zorder>
     <zorder>pbFPS3</zorder>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>Color 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="color_widgets::ColorSelector" name="csColor1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>Color 2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="color_widgets::ColorSelector" name="csColor2"/>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>200</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>color_widgets::ColorPreview</class>
   <extends>QWidget</extends>
   <header>color_preview.hpp</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorSelector</class>
   <extends>color_widgets::ColorPreview</extends>
   <header>color_selector.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
